terraform/gcp/juniper_alerts.tf

```diff
@@ -1 +1,46 @@
-# todo: alert on application errors
+# creates alerts on the juniper-dev namespace
+# for application errors.
+
+# create a logging metric to count errors. if you want to
+# filter out errors, you can edit the filter.
+resource "google_logging_metric" "juniper_app_error_logging_metric" {
+  name   = "juniper-error/metric"
+  filter = <<EOT
+severity>=ERROR AND resource.labels.namespace_name="${var.k8s_namespace}" AND (resource.labels.container_name = "admin" OR resource.labels.container_name = "participant")
+EOT
+  metric_descriptor {
+    metric_kind = "DELTA"
+    value_type  = "INT64"
+  }
+}
+
+# NOTE: ANYTIME YOU EDIT THIS, MAKE SURE YOU DISABLE
+#       "notify on incident closure" MANUALLY VIA THE UI
+#        There is no way to disable this via terraform
```

Comment:
as of ~1 month ago there seems to have been some movement on this but it hasn't landed officially yet